app/controllers/tickets_controller.rb

```diff
@@ -33,7 +32,35 @@ def buy
     redirect_to root_path
   end
 
+  def edit
+    @events = Event.all
+  end
+
   def update
+    parameters = params.require(:ticket).permit(:place, :event_id, :price)
+
+    if @ticket.update(parameters)
+      flash[:notice] = "Your ticket has been edited."
+    else
+      flash[:notice] = "Your ticket hasn't been edited."
```

Comment:
Czy to, że bilet nie został zedytowany to jest błąd, czy zwykłe, oczekiwane zachowanie? jeśli błąd to chyba lepiej użyć klucza `:alert`, zamiast `notice`.